src/jukebox/run_configure_audio.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+
+import pulsectl
+
+import misc.inputminus as pyil
+from misc.simplecolors import Colors
+import jukebox.cfghandler
+
+deliminator_length = 79
+
+
+def msg_highlight(msg, color=Colors.lightblue):
+    print(f"\n{color}" + "*" * deliminator_length)
+    print(msg)
+    print("*" * deliminator_length + f"{Colors.reset}")
+
+
+def check_jukebox_running():
+    pass
+
+
+def query_sinks():
+    pulse = pulsectl.Pulse('jukebox-config')
+    sinks = pulse.sink_list()
+    msg_highlight('Available audio outputs')
+    for idx, sink in enumerate(sinks):
+        print(f"{Colors.lightgreen}{idx:2d}{Colors.reset}:"
+              f"  {Colors.lightcyan}{sink.name}{Colors.reset}")
+    print("")
+    primary_idx = pyil.input_int("Primary audio output (no bluetooth)?", min=0, max=len(sinks) - 1,
+                                 prompt_color=Colors.lightgreen,
+                                 prompt_hint=True, blank=0)
+    primary = sinks[primary_idx].name
+    print(f"Primary audio output = {primary}\n")
+    secondary_idx = pyil.input_int("Secondary audio output (typically bluetooth)? Set to -1 for empty.",
+                                   min=-1, max=len(sinks) - 1,
+                                   prompt_color=Colors.lightgreen,
+                                   prompt_hint=True, blank=-1)
+    secondary = None
+    toggle_on_connect = False
+    if secondary_idx >= 0:
+        secondary = sinks[secondary_idx].name
+        print(f"Secondary audio output = {secondary}\n")
+        toggle_on_connect = pyil.input_yesno("Automatically toggle output on connection of secndary device?",
+                                             prompt_color=Colors.lightgreen,
+                                             prompt_hint=True, blank=True)
+    return primary, secondary, toggle_on_connect
+
+
+def configure_jukebox(filename, primary, secondary, toggle_on_connect):
+    cfg_jukebox = jukebox.cfghandler.get_handler('juke')
+    cfg_jukebox.load(filename)
+
+    cfg_jukebox.setn('pulse', 'toggle_on_connect', value=toggle_on_connect)
+
+    cfg_jukebox.setn('pulse', 'outputs', value={})
+    key = 'primary'
+    cfg_jukebox.setn('pulse', 'outputs', key, 'alias', value='Speakers')
+    cfg_jukebox.setn('pulse', 'outputs', key, 'volume_limit', value=100)
+    cfg_jukebox.setn('pulse', 'outputs', key, 'soft_max_volume', value=100)
+    cfg_jukebox.setn('pulse', 'outputs', key, 'pulse_sink_name', value=primary)
+
+    if secondary is not None:
+        key = 'secondary'
+        cfg_jukebox.setn('pulse', 'outputs', key, 'alias', value='Bluetooth Headset')
+        cfg_jukebox.setn('pulse', 'outputs', key, 'volume_limit', value=100)
+        cfg_jukebox.setn('pulse', 'outputs', key, 'soft_max_volume', value=100)
+        cfg_jukebox.setn('pulse', 'outputs', key, 'pulse_sink_name', value=secondary)
+
+    cfg_jukebox.save()
+
+
+def welcome():
+    msg_highlight('The Jukebox audio output configuration tool')
+    print("""
+Please note:
```

Comment:
Should we put this info also into the docs? I know some of it is there, but not its clarity!